src/objective-c/tests/GRPCClientTests.m

```diff
@@ -303,7 +303,7 @@ - (void)testExceptions {
 
   // Try to set parameters to nil for GRPCCall. This should cause an exception
   @try {
-    GRPCCall *call = [[GRPCCall alloc] initWithHost:nil
+    GRPCCall *call __unused = [[GRPCCall alloc] initWithHost:nil
```

Comment:
The object gets initialized even if you don't assign the result of `init` to a variable.